jobs/JGLOBAL_WAVE_PREP

```diff
@@ -86,7 +86,7 @@ else
   fi 
   if [ $WW3ICEINP = "YES" ]; then 
     if [ ! -L $ROTDIR/${CDUMP}.${PDY}/${cyc}/atmos/${WAVICEFILE} ]; then # Check if symlink already exists in ROTDIR
-      $NLN $DMPDIR/$CDUMP.${PDY}/$cyc/${WAVICEFILE} $ROTDIR/$CDUMP.${PDY}/$cyc/atmos/${WAVICEFILE}
+      $NLN $DMPDIR/$CDUMP.${PDY}/$cyc/atmos/${WAVICEFILE} $ROTDIR/$CDUMP.${PDY}/$cyc/atmos/${WAVICEFILE}
```

Comment:
I must've missed it the first time, but why is a wave/ice file being put under 'atmos'?

Reply:
It's the ice analysis file that the wave model uses.